modules/network/arbitrum.ts

```diff
@@ -175,6 +175,25 @@ const arbitrumNetworkData: NetworkData = {
                 path: 'sfrxethApr',
                 isIbYield: true,
             },
+            sFRAX: {
+                tokenAddress: '0xe3b3fe7bca19ca77ad877a5bebab186becfad906',
+                sourceUrl: 'https://api.frax.finance/v2/frax/sfrax/summary/history?range=1d',
+                path: 'items.0.sfraxApr',
+                isIbYield: true,
+            },
+            ankrETH: {
+                tokenAddress: '0xe05a08226c49b636acf99c40da8dc6af83ce5bb3',
+                sourceUrl: 'https://api.staking.ankr.com/v1alpha/metrics',
+                path: 'services.{serviceName == "eth"}.apy',
+                isIbYield: true,
+            },
+            plsRDNT: {
+                tokenAddress: '0x6dbf2155b0636cb3fd5359fccefb8a2c02b6cb51',
+                sourceUrl: 'https://plutusdao.io/api/getPlsRdntInfo',
```

Comment:
is the cloudflare captcha fixed?

Reply:
Shall we remove this until we receive confirmation it's working?

Reply:
it's ready to go